content/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll.md

```diff
@@ -51,9 +51,17 @@ Before you can use Jekyll to test a site, you must:
 
    {% note %}
 
-   **Note:** If you've installed Ruby 3.0 or later (which you may have if you installed the default version via Homebrew), you might get an error at this step. That's because these versions of Ruby no longer come with `webrick` installed.
+   **Notes:**
+   - If you've installed Ruby 3.0 or later (which you may have if you installed the default version via Homebrew), you might get an error at this step. That's because these versions of Ruby no longer come with `webrick` installed.
+
+     To fix the error, try running `bundle add webrick`, then re-running `bundle exec jekyll serve`.
+
+   - If your `_config.yml` file's `baseurl` field contains your GitHub repository's link, you can use the following command to run the site locally without changing the `baseurl` in the `_config.yml` file:
```

Comment:
Great tip! I have a minor suggestion on wording to help connect why you'd want to do this when running locally:

```suggestion
   - If your `_config.yml` file's `baseurl` field contains your GitHub repository's link, you can use the following command when building locally to ignore that value and serve the site on `localhost:4000/`:
```

Reply:
That's a great suggestion @tcbyrd 
It does help clarify how and why the command is useful.